vips/vip-193/vip-193.ts

```diff
@@ -1,26 +1,17 @@
 import { ProposalType } from "../../src/types";
 import { makeProposal } from "../../src/utils";
-import commands192 from "../vip-192/commands";
-import commands from "./commands";
+import commands from "../vip-192/commands";
```

Comment:
i think we have already executed these commands in previous vip(vip-192), can you please elaborate the sense of executing the same in current vip(vip-193).